src/core/ready-no-deferred.js

```diff
@@ -0,0 +1,109 @@
+define( [
+	"../core",
+	"../var/document"
+], function( jQuery, document ) {
+
+var readyCallbacks = [],
+	readyFiring = false,
+	whenReady = function( fn ) {
+		readyCallbacks.push( fn );
+	},
+	executeReady = function( fn ) {
+
+		// Prevent errors from freezing future callback execution (gh-1823)
+		// Not backwards-compatible as this does not execute sync
+		window.setTimeout( function() {
+			fn.call( document, jQuery );
```

Comment:
Perhaps we can leave that subject to change/undocumented. I don't want to verify it if we're going to change it in a future version.